omnigibson/robots/behavior_robot.py

```diff
@@ -142,10 +142,6 @@ def __init__(
     def usd_path(self):
         return os.path.join(gm.ASSET_PATH, "models/behavior_robot/usd/BehaviorRobot.usd")
 
-    @property
-    def model_name(self):
-        return "BehaviorRobot"
```

Comment:
What's the reason for removing this?

Reply:
it's automatically generated in the robot base class!